code_gen_tester/pubspec.yaml

```diff
@@ -1,13 +1,13 @@
 name: code_gen_tester
 version: 0.0.1
 environment:
-    sdk: '>=2.0.0-dev.68.0 <3.0.0'
+    sdk: '>=2.1.0 <3.0.0'
 dependencies:
```

Comment:
nit: do you think you could upgrade the other dependencies too? like build_runner

It should be straightforward. But if it's not, don't worry too much about it.

Reply:
I'm fine doing that, but referring to my other comment, I want to discuss about latest dependency vs backward compatibility for other developers.

Reply:
code_gen_tester is internal to functional_widget so that shouldn't matter in this specific case